src/gesture-handler/defaults.ts

```diff
@@ -16,6 +17,13 @@ export const DefaultEvent: GestureUpdateEvent<PanGestureHandlerEventPayload> = {
   velocityY: 0,
   x: 0,
   y: 0,
+
+  // These two were added in recent versions of gesture handler
+  // and they are required to specify. This should be backward
+  // compatible unless they strictly parse the objects, which seems
+  // not likely. PointerType is present since 2.16.0, StylusData since 2.20.0
+  pointerType: PointerType.TOUCH,
+  stylusData: undefined,
```

Comment:
Reinstalling root-level deps caused `gesture-handler` to update, which has recently added these two non-optional fields to the `GestureUpdateEvent` and `PanGestureHandlerEventPayload` types. I've consulted with @m-bert and `PointerType.TOUCH` should be fine for our usecases. 

`stylusData` is defined in such way that the prop must be passed, even if you specify it as undefined. I believe this should be changed in `gesture-handler` from `stylusData: StylusData | undefined` to `stylusData?: StylusData | undefined`